models/cow_protocol/tx_hash_labels/cow_protocol_tx_hash_labels_all_schema.yml

```diff
@@ -0,0 +1,36 @@
+version: 2
+
+models:
+  - name: cow_protocol_tx_hash_labels_all
+    tests:
+      - dbt_utils.unique_combination_of_columns:
+          combination_of_columns:
+            - tx_hash
+            - name
+            - category
+            - blockchain
```

Comment:
Is this a generic test framework that should just be applied here, or so we have to update something in the test files?

Reply:
It's something I've found inside the `labels_all_schema.yml` and assumed we would want the same test for our own table.